submissions/aymkin/OOP-JS (tjsw)/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+
+Code repository: https://github.com/aymkin/a-tiny-JS-world/tree/tiny-js-world-oop-aymkin
+Web app: https://aymkin.github.io/a-tiny-JS-world/
+   */
+
+function Inhobitant(species, name, gender, friends, say) {
+	this.species = species;
+	this.name = name;
+	this.gender = gender;
+	this.friends = friends;
+	this.say = say;
+}
+
+Inhobitant.prototype.makeSound = function() {
+	return this.say;
+};
+
+Inhobitant.prototype.getFriend = function() {
+	return this.hasOwnProperty('friends')
```

Comment:
Class `Inhobitant` has own property `friends` even if the value for it is not defined. Under current implementation the `else` condition of the ternary operator will not fire ever.
Please, fix analyzing the `.friends` value.

Reply:
add the checks if the friend array is empty